internal/repository/repository_contract.go

```diff
@@ -5,6 +5,7 @@ import "time"
 type FileRepository interface {
 	FindByIdentifier(identifier string) (*FileModel, error)
 	Save(p SaveFileParam) error
+	Delete(identifier string) error
```

Comment:
I think it's better for repository to be specific so we don't need to refactor the param in the future when there's other `Delete` usecase, e.g: `DeleteByIdentifier(identifier string) error`

Or use struct as a param, e.g: `Delete(p DeleteFileParam) error`